docs/api/session.md

```diff
@@ -906,15 +906,19 @@ win.webContents.session.setCertificateVerifyProc((request, callback) => {
     * `speaker-selection` - Request to enumerate and select audio output devices via the [speaker-selection permissions policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection).
     * `window-management` - Request access to enumerate screens using the [`getScreenDetails`](https://developer.chrome.com/en/articles/multi-screen-window-placement/) API.
     * `unknown` - An unrecognized permission request.
+    * `fileSystem` - Request access to read, write, and file management capabilities using the [File System API](https://developer.mozilla.org/en-US/docs/Web/API/File_System_API).
   * `callback` Function
     * `permissionGranted` boolean - Allow or deny the permission.
   * `details` Object - Some properties are only available on certain permission types.
     * `externalURL` string (optional) - The url of the `openExternal` request.
     * `securityOrigin` string (optional) - The security origin of the `media` request.
     * `mediaTypes` string[] (optional) - The types of media access being requested, elements can be `video`
-      or `audio`
-    * `requestingUrl` string - The last URL the requesting frame loaded
-    * `isMainFrame` boolean - Whether the frame making the request is the main frame
+      or `audio`.
+    * `requestingUrl` string - The last URL the requesting frame loaded.
+    * `isMainFrame` boolean - Whether the frame making the request is the main frame.
+    * `path` string (optional) - The path of the `file-system` request.
+    * `isDirectory` boolean (optional) - Whether the `file-system` request is a directory.
+    * `accessType` string (optional) - The access type of the `file-system` request. Can be `writable` or `readable`.
```

Comment:
A little concerned with genericness of these names having a future conflict with another new request type. Will defer to API WG on best way to address it, just wanted to flag the concern.

Reply:
Agreed - this is a good point and some of the nomenclature can be fairly trivially clarified i think.

Reply:
updated this a bit: 

`path` -> `filePath`
`accessType` -> `fileAccessType`